src/js/hca-rjsf/config/form.js

```diff
@@ -670,31 +672,50 @@ const formConfig = {
             'view:spouseIncome': {
               'ui:title': 'Spouse income',
               'ui:options': {
-                hideIf: (formData) => formData.maritalStatus !== 'married' // Something else too?
+                hideIf: (formData) => !formData.maritalStatus || (formData.maritalStatus.toLowerCase() !== 'married') // Something else too?
               },
               spouseGrossIncome: {
                 'ui:title': 'Spouse gross annual income from employment',
-                'ui:required': (formData) => formData.maritalStatus === 'married'
+                'ui:required': (formData) => formData.maritalStatus === 'Married'
               },
               spouseNetIncome: {
                 'ui:title': 'Spouse Net Income from your Farm, Ranch, Property or Business',
-                'ui:required': (formData) => formData.maritalStatus === 'married'
+                'ui:required': (formData) => formData.maritalStatus === 'Married'
               },
               spouseOtherIncome: {
                 'ui:title': 'Spouse Other Income Amount',
-                'ui:required': (formData) => formData.maritalStatus === 'married'
+                'ui:required': (formData) => formData.maritalStatus === 'Married'
               }
             },
-            'view:childrenIncome': {
+            children: {
+              // 'ui:title': 'Child income',
+              'ui:field': 'BasicArrayField',
+              items: _.merge(childIncomeUiSchema, {
+                'ui:options': {
+                  updateSchema: (formData, schema, uiSchema, index) => {
```

Comment:
Because I skipped it while refactoring....No reason. It should be in there. I'm on it!